Test/dafnydoc/doc1/docs-expected/TestModule.B_1070997150.html

```diff
@@ -0,0 +1,29 @@
+
+<!doctype html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=1000px, initial-scale=1">
```

Comment:
It's very unusual to me to see an explicit width.
Couldn't you use the normal one?

`<meta name="viewport" content="width=device-width, initial-scale=1">`

Reply:
Changed (The original came from a online example)